device-data/types/bolus/dual-square.md

```diff
@@ -60,7 +60,7 @@ We plan to migrate all Tidepool data to use `combo` as the value of this sub-typ
 		platform: yes
 	Numerical type: Floating point value rounded to the appropriate significant figures for the device's precision.
 	Range:
-		min: > 0.0
+		min: >= 0.0
```

Comment:
Is it possible to not include auto-generated files in the repo, or make them non-editable, or move user-editable sections to a different file? Because I have a feeling we may have a quite a few changes steam-rolled in future if we're not careful ;)